deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,361 @@
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/cluster"
+                  }
+                },
+                "name": "prometheus-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/operator"
+                  }
+                },
+                "name": "prometheus-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "parameters": [
+                    {
+                      "name": "s3_endpoint_url",
+                      "value": "s3.odh.com"
+                    }
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "jupyterhub/jupyterhub"
+                  }
+                },
+                "name": "jupyterhub"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "additional"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "jupyterhub/notebook-images"
+                  }
+                },
+                "name": "notebook-images"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "airflow/operator"
+                  }
+                },
+                "name": "airflow-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "airflow/cluster"
+                  }
+                },
+                "name": "airflow-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/cluster"
+                  }
+                },
+                "name": "odhargo-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/odhargo"
+                  }
+                },
+                "name": "odhargo"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odh-dashboard"
+                  }
+                },
+                "name": "odh-dashboard"
+              }
+            ],
+            "repos": [
+              {
+                "name": "kf-manifests",
+                "uri": "https://github.com/opendatahub-io/manifests/tarball/v1.3-openshift"
+              },
+              {
```

Comment:
should kf-manifest point to 1.4 branch now?

Reply:
Updated